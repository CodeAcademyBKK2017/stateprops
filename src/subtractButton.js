import React, { Component } from 'react';

class Subtract extends Component {
  render() {
    return (

    )
  }
}

export default Subtract
