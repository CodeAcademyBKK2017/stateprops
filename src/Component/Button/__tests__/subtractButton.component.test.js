import React from 'react';
import subtractButton from '../subtractButton.component';
import renderer from 'react-test-renderer';

test('subtractButton Snapshot Test',() => {
  let text = 'Subtract';
  const tree = renderer.create(<subtractButton onClick={jest.fn()}>{text}</subtractButton>);
  expect(tree).toMatchSnapshot();
});