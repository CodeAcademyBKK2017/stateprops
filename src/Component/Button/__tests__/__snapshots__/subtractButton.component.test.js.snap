// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`subtractButton Snapshot Test 1`] = `
<subtractButton
  onClick={[Function]}
>
  Subtract
</subtractButton>
`;
